pallene/coder.lua

```diff
@@ -968,7 +968,7 @@ gen_cmd["Binop"] = function(self, cmd, _func)
     elseif op == "FltSub"    then return binop("-")
     elseif op == "FltMul"    then return binop("*")
     elseif op == "FltDivi"   then return flt_divi()
-    elseif op == "FltMod"    then return flt_mod()
+    elseif op == "FltMod"    then return flt_mod("pallene_float_modi")
```

Comment:
The flt_mod function doesn't receive any parameters, so there is no need to pass a parameter here.

Reply:
Ah, yeah that's right. Things are starting to make more sense now!